engine/runtime/src/main/scala/org/enso/compiler/data/BindingsMap.scala

```diff
@@ -169,7 +173,8 @@ case class BindingsMap(
   }
 
   private def findExportedCandidatesInImports(
-    name: String
+    name: String,
+    caseSensitive: Boolean
```

Comment:
What needs this? I can't think of a use case for `caseSensitive=false`?

Reply:
leftover, thanks for noticing